docs/handbook/navigation-board.md

```diff
@@ -3,45 +3,6 @@ title: 🗺️ State of Raids
 ---
 # Ongoing Raids
 
-Find what interests you then join by following the instructions below or simply contacting the person connected to the raid, through [Discord](https://discord.gg/Hf54gd8).
+Find what interests you then join by following the links or simply contacting the person connected, through [Discord](https://discord.gg/Hf54gd8).
 
 <iframe width="1024" height="1024" src="https://miro.com/app/embed/o9J_ksFmLEI=/?" frameborder="0" scrolling="no" allowfullscreen></iframe>
-
-## XP & Seed DAO
```

Comment:
Why delete all this stuff? Seems like a lot of people found it valuable

Reply:
because it's written directly on the board now, will make PRs more descriptive in the future 😬